### PR TITLE
Visualizations (with more comments)

### PR DESCRIPTION
**Pull request by derpyplops**
Current usage
elk plot plots your most recent sweep
elk plot --sweep [sweep_name] to visualize particular sweep
elk sweep ... --visualize visualizes after sweeping

Future PRs
elk plot --sweep [sweep_1 .. sweep_n] plots other sweeps
data validation per plot
tests
elk plot --elicit

### CODE REVIEW

File: tests/test_viz.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import unittest
```
[the remark this answers is not in the document]
Comment (derpyplops) [reply tests/test_viz.py:1]: oh right ok can

File: elk/plotting/command.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+from ..files import sweeps_dir
+from .visualize import visualize_sweep
+
+
+def pretty_error(msg):
+    rich.print(Panel(f"[red]Error[/red]: {msg}"))
+
+
+@dataclass
+class Plot:
```
Comment (lauritowal) [inline elk/plotting/command.py:17]: add description of class

File: elk/plotting/command.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            )
+        elif (sweep / "viz").exists() and not self.overwrite:
+            pretty_error(
+                f"[blue]{sweep / 'viz'}[/blue] already exists. "
+                f"Use --overwrite to overwrite."
+            )
+        else:
+            if self.overwrite:
+                shutil.rmtree(sweep / "viz")
+            visualize_sweep(sweep)
```
Comment (norabelrose) [inline elk/plotting/command.py:42]: This doesn't actually use the value of `self.sweeps` at all, so this command _only_ supports visualizing the most recent sweep right now
Captura de pantalla 2023-05-22 a la(s) 11 53 53 a m

Comment (norabelrose) [reply elk/plotting/command.py]: Fixed

**Comment by norabelrose** [comment]
Major bug: This command currently just save the same multiplots for every model; I think it chooses the first model or something and then saves that N times. Working on fixing now

File: elk/plotting/command.py [hunk 4]
```diff
@@ -2,17 +2,16 @@
...
 from simple_parsing import field
 
 from ..files import sweeps_dir
+from ..utils import colorize
 from .visualize import visualize_sweep
 
 
 def pretty_error(msg):
     """Prints a pretty error message."""
-    rich.print(Panel(f"[red]Error[/red]: {msg}"))
```
Comment (derpyplops) [inline elk/plotting/command.py]: @norabelrose Could you explain why you changed this?

File: elk/plotting/visualize.py [hunk 5]
```diff
@@ -1,9 +1,6 @@
-from __future__ import annotations
```
Comment (derpyplops) [inline elk/plotting/visualize.py]: @norabelrose Could you explain why you removed this? I added this import following the advice of https://stackoverflow.com/questions/33533148/how-do-i-type-hint-a-method-with-the-type-of-the-enclosing-class

File: elk/plotting/visualize.py [hunk 6]
```diff
@@ -7,8 +7,8 @@
 import plotly.graph_objects as go
 from plotly.colors import qualitative
 from plotly.subplots import make_subplots
-
-from .utils import display_table
```
Comment (derpyplops) [inline elk/plotting/visualize.py]: Could you explain why you changed this? Do you prefer having less files rather than more?

**Review by lauritowal — APPROVED** [review]
okay tested it after Nora's changes and looks good to me

**Pull request merged**